help/destinations/catalog/adobe/marketo-engage-person-sync.md

```diff
@@ -0,0 +1,120 @@
+| Segmentation Service |     ✓     | Audiences generated through the Experience Platform [Segmentation Service](https://experienceleague.adobe.com/en/docs/experience-platform/segmentation/home). |
+| Custom uploads       |     ✓     | Audiences imported into Experience Platform from CSV files.                                                                                                   |
+
+## Export Type and Frequency {#export-type-and-frequency}
+
+Refer to the table below for information about the destination export type and frequency.
+
+| Item             | Type      | Notes                                                                                                                                                                                                                                                                                                                                                                                     |
+| ---------------- | --------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Export frequency | Streaming | Streaming destinations are “always on” API-based connections. As soon as a profile is updated in Experience Platform based on audience evaluation, the connector sends the update downstream to the destination platform. Read more about [streaming destinations](/help/destinations/destination-types#streaming-destinations). |
+
+{style="table-layout:auto"}
+
+## Set up Destination {#set-up-destination}
+
+>[!IMPORTANT]
+>
+>* To connect to the destination, you need the **[!UICONTROL View Destinations]** and **[!UICONTROL Manage Destinations]** [access control permissions](/help/access-control/home.md#permissions).
+
+To configure a Marketo Engage instance as a destination, that instance must be in the same IMS org as your CDP instance. If you have already configured a destination, you may select an existing Marketo account to use with your new configuration. If not, click the Connector to Destination prompt, which will allow you to set the name, description, and Marketo Munchkin ID of the desired destination. Your Marketo Instance's Munchkin ID can be found in the Admin->Munchkin menu.
+
+>[!IMPORTANT]
+>
+>The user setting up the destination must have [Edit Person](https://experienceleague.adobe.com/en/docs/marketo/using/product-docs/administration/users-and-roles/descriptions-of-role-permissions#access-database) permission in the Marketo instance and partition.
+
+![Connect to Destination](../../assets/catalog/adobe/marketo-engage-person-sync/connect-to-destination.png)
```

Comment:
please call out and describe all these selection fields like Munchkin IDs etc as a bullet list, similar to what we do in the template https://experienceleague.adobe.com/en/docs/experience-platform/destinations/destination-sdk/document-destination/self-service-template#destination-details

This way, the fields and their descriptions stand out to the reader.